site/releases/2023/2023-aug-15/highlights.qmd

```diff
@@ -13,36 +15,176 @@ This release includes a number of improvements for the developer experience when
 --->
 
 <!---John6797/sc 1980/metric test class dx improvements by @johnwalz97 in [#220](https://github.com/validmind/validmind-python/pull/220)--->
-- **Improved developer experience for the Metric test class**. We made a number of enhancements to the {{< var validmind.developer >}} to improve the experience for developers:
+#### Improved developer experience for the Metric test class
 
-   * **Better test organization**: Added a `metadata` and `tags` attribute to the `Test` base class for better categorization and filtering of tests.
-   * **Better filtering**: Added a new `task_type` argument to the `list_tests()` function to allow simple filtering by `task_type` set in a test.
-   * **Clearer naming**: Renamed `required_context` to `required_inputs` across the whole ValidMind SDK.
+We made a number of enhancements to the {{< var validmind.developer >}} to improve the experience for developers.
+
+##### Better test organization
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+Added a `metadata` and `tags` attribute to the `Test` base class for better categorization and filtering of tests.
+:::
+
+::: {.w-20-ns}
+<form method="get" action="/validmind/validmind/vm_models.html#Test" target="_blank">
```

Comment:
This PR adds **59 of these buttons**. That's a lot of buttons! I'm not opposed to that in principle, but the original intent was to use them very sparingly to make sure they pop. I don't think they are effective at scale ... In which case we might consider switching over to a less loud Tachyons button.